cmd/build/helmify/static/values.yaml

```diff
@@ -53,15 +53,15 @@ postInstall:
     enabled: true
     image:
       repository: openpolicyagent/gatekeeper-crds
-      tag: v3.9.0-beta.2
+      tag: v3.9.0-rc.1
       pullPolicy: IfNotPresent
       pullSecrets: []
     extraNamespaces: []
   probeWebhook:
     enabled: true
     image:
       repository: curlimages/curl
-      tag: 7.83.1
+      tag: v3.9.0-rc.1
```

Comment:
Looks like this is not supposed to be change

Reply:
It's caused by https://github.com/open-policy-agent/gatekeeper/blob/master/Makefile#L395. I just pushed a commit to fix the tag and I can open another PR to fix this.